presidio-analyzer/analyzer/app_tracer.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+
+class AppTracer:
+    """This class handles app traces"""
```

Comment:
This is also affecting the logger, isn't it? Maybe we should combine the two and add a 'trace' call to the logger (We can potentially have two instances of this, but it's subject to making sure that the logger isn't a singleton or something like that)